docs/clients/cocoa/dsym.rst

```diff
@@ -132,7 +132,10 @@ Shell: `/bin/bash`
     export SENTRY_ORG=___ORG_NAME___
     export SENTRY_PROJECT=___PROJECT_NAME___
     export SENTRY_AUTH_TOKEN=YOUR_AUTH_TOKEN
-    sentry-cli upload-dsym
+    ERROR=$(sentry-cli upload-dsym 2>&1 >/dev/null)
+    if [ ! -z "$ERROR" ]; then
```

Comment:
so I played around with a few other ways of capturing exit codes and stderr/stdout ...

We only want the shell script to output something when `sentry-cli` fails, and then we want to capture the `error` and show it as a warning in xcode ...
If we let `sentry-cli` return the exit code without capturing, the xcode build will fail because a shell script produced an error ...

So I would propose we leave it like this